pkg/proxy/proxier.go

```diff
@@ -300,12 +301,13 @@ func newProxySocket(protocol api.Protocol, ip net.IP, port int) (proxySocket, er
 // Proxier is a simple proxy for TCP connections between a localhost:lport
 // and services that provide the actual implementations.
 type Proxier struct {
-	loadBalancer LoadBalancer
-	mu           sync.Mutex // protects serviceMap
-	serviceMap   map[string]*serviceInfo
-	listenIP     net.IP
-	iptables     iptables.Interface
-	hostIP       net.IP
+	loadBalancer  LoadBalancer
+	mu            sync.Mutex // protects serviceMap
+	serviceMap    map[string]*serviceInfo
+	numProxyLoops int32 // use atomic ops to access this
```

Comment:
Can you note that this is only used for testing (and in the waitFor... function too)